modules/Makefile.circleci

```diff
@@ -42,11 +42,13 @@ circle\:release:
 ## Deploy to kubernetes after obtaining an exclusive lock
 circle\:deploy-kubernetes:
 	$(call assert,CIRCLE_BRANCH)
+	$(SELF) circle:tag kubernetes:info
 	@if [ -z "$(CIRCLE_TOKEN)" ]; then \
-    echo -e "$(call red,WARN:) Deploying $(RELEASE) without obtaining lock; CIRCLE_TOKEN not defined"; \
-    $(SELF) circle:tag kubernetes:info kubernetes:deploy; \
+	echo -e "$(call red,WARN:) Deploying $(RELEASE) without obtaining lock; CIRCLE_TOKEN not defined"; \
+	$(SELF) kubernetes:deploy; \
   else \
 	  echo "INFO: Deploying $(RELEASE)"; \
-	  $(MAKEFILE_DIR)/bin/circle-do-exclusively.sh --branch $(CIRCLE_BRANCH) $(SELF) circle:tag kubernetes:info kubernetes:deploy; \
+	  $(MAKEFILE_DIR)/bin/circle-do-exclusively.sh --branch $(CIRCLE_BRANCH) $(SELF) kubernetes:deploy; \
   fi
+	$(SELF) kubernetes:list:rc kubernetes:list:svc
```

Comment:
not sure if this level of hierarchy is needed? maybe just `kubernetes:list-rc`

e.g. we have 
`circle:deploy-kubernetes`, while this could technically be `circle:deploy:kubernetes`, i thought it was too much.